src/modules/number/index.ts

```diff
@@ -421,17 +431,40 @@ export class NumberModule extends SimpleModuleBase {
 
     const min = BigInt(options.min ?? 0);
     const max = BigInt(options.max ?? min + BigInt(999999999999999));
+    const multipleOf = BigInt(options.multipleOf ?? 1);
 
-    if (max === min) {
-      return min;
+    if (max < min) {
+      throw new FakerError(`Max ${max} should be larger than min ${min}.`);
     }
 
-    if (max < min) {
-      throw new FakerError(`Max ${max} should be larger then min ${min}.`);
+    if (multipleOf <= BigInt(0)) {
+      throw new FakerError(`multipleOf should be greater than 0.`);
+    }
+
+    const effectiveMin =
+      min % multipleOf === BigInt(0)
+        ? min / multipleOf
+        : BigInt(0) <= min
+          ? min / multipleOf + BigInt(1)
+          : min / multipleOf;
+    const effectiveMax =
+      max % multipleOf === BigInt(0)
+        ? max / multipleOf
+        : BigInt(0) <= max
+          ? max / multipleOf
+          : max / multipleOf - BigInt(1);
```

Comment:
IMO this can be simplified a bit (and extended with a comment for why we do it):

```suggestion
        const effectiveMin = min / multipleOf + (min % multipleOf > 0n ? 1n : 0n); // Math.ceil(min / multipleOf)
        const effectiveMax = max / multipleOf - (max % multipleOf < 0n ? 1n : 0n); // Math.floor(max / multipleOf)
```